pkg/kubelet/volumemanager/volume_manager.go

```diff
@@ -375,9 +376,10 @@ func (vm *volumeManager) WaitForAttachAndMount(pod *v1.Pod) error {
 		}
 
 		return fmt.Errorf(
-			"timeout expired waiting for volumes to attach or mount for pod %q/%q. list of unmounted volumes=%v. list of unattached volumes=%v",
+			"failed to attach or mount for pod %q/%q: %s. List of unmounted volumes=%v, list of unattached volumes=%v.",
 			pod.Namespace,
 			pod.Name,
+			err,
```

Comment:
For readability in multiple errors case, would it be better to output errors after lists of volumes?
(It might be too much to add "List of errors=" before errors, though.)

Reply:
Ack, I reordered the messages as you suggest:

```
Events:
  Type     Reason       Age   From                Message
  ----     ------       ----  ----                -------
  Warning  FailedMount  1s    kubelet, 127.0.0.1  Unable to attach or mount 
volumes: unmounted volumes=[vol], unattached volumes=[vol default-token-p67zc]: 
error processing PVC default/myclaim: failed to fetch PVC from API server: 
persistentvolumeclaims "myclaim" not found
```